frontend/src/metabase/common/hooks/use-notification-channels/use-notification-channels.ts

```diff
@@ -0,0 +1,15 @@
+import { useGetChannelInfoQuery } from "metabase/api";
+
+export const useHasAnyNotificationChannel = (): boolean => {
```

Comment:
I think I've noticed us using this pattern more lately (creating hooks to determine things we would have probably written selectors for in the past). It makes sense as we rely more and more on RTK Query. Nothing really needs to change here, but calling out that if we think we'll have a library of metabase specific hooks, we may need to get better about organizing them instead of throwing them all in `common/hooks`

Reply:
probably a good frontend guild topic. The hooks are way better because they fetch data when they need it where, in a lot of places in the past we just hoped/assumed certain state was in the redux store already.